rllib/algorithms/impala/torch/impala_torch_policy_rlm.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from typing import Dict
+
+from ray.rllib.algorithms.ppo.ppo_torch_policy import validate_config
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.policy.torch_mixins import (
+    EntropyCoeffSchedule,
+    LearningRateSchedule,
+)
+from ray.rllib.policy.torch_policy_v2 import TorchPolicyV2
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.utils.torch_utils import (
+    explained_variance,
+    global_norm,
+)
+from ray.rllib.utils.typing import TensorType
+
+torch, nn = try_import_torch()
+
+logger = logging.getLogger(__name__)
+
+
+class ImpalaTorchPolicyWithRLModule(
```

Comment:
We don't need a loss function here thanks to our automatic inference of view requirements.

Reply:
algorithm implementors don't need one, but in order to merge this, I think it would be better to implement one now, use it to test the learner loss, and advertise it since in 2.4 we won't be advertising the learners.

Reply:
Done